django_netjsongraph/base/node.py

```diff
@@ -114,9 +81,11 @@ def get_from_address(cls, address, topology):
         :param topology: Topology instance
         :returns: Node object or None
         """
-        address = ';{0};'.format(address)
-        return cls.objects.filter(topology=topology,
-                                  addresses__contains=address).first()
+        objs = list(cls.objects.filter(addresses__contains=address))
+        for obj in objs:
+            if address in obj.addresses:
+                return obj
+        return None
```

Comment:
replace this method code with:

```python
address = '"{}"'.format(address)
return cls.objects.filter(topology=topology,
                          addresses__contains=address).first()
```

For the same reasons explained previously.

PS: for some reason you have also removed the filtering by topology, I think that was unintended, it shouldn't be done.